### PR TITLE
Add Pyannote VAD pipelines

### PR DESCRIPTION
**Pull request by ShiyangLai**
Pyannote VAD pipeline

### CODE REVIEW

File: pipelines/vad/utils.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+"""
+
+import random
+import numpy as np
+import pandas as pd
+from scipy import ndimage
+import yaml
+
+
+def generate_ramdom_code():
```
Comment (graziul) [inline pipelines/vad/utils.py:14]: "generate_ramdom_code" meant to be "generate_random_code"? Unsure if this breaks anything anywhere but assumed a typo.

**Review by graziul — CHANGES_REQUESTED** [review]
Please remove transcripts from one notebook - looks great otherwise!

File: pipelines/vad/utils.py (hunk 1 above)
Comment (ShiyangLai) [reply pipelines/vad/utils.py:14]: This function is to generate an identical reference id for each assigned model training job which is different from the job id that is automatically assigned by computing clusters. The ids can be used to gain the logged training hyperparameters and the evaluation results of corresponding model training jobs. Sorry for the confusing name of the function, I will update it.

Comment (graziul) [reply pipelines/vad/utils.py:14]: Okay, understood and thank you for explaining.

File: pipelines/notebooks/bpc_vad_error.ipynb [hunk 2]
```diff
@@ -0,0 +1,1295 @@
...
+       "    <tr>\n",
+       "      <th>2921</th>\n",
+       "      <td>Zone1</td>\n",
+       "      <td>/project/graziul/data/Zone1/2018_08_10/2018081...</td>\n",
+       "      <td>2018_08_10</td>\n",
+       "      <td>536595</td>\n",
+       "      <td>4.771</td>\n",
+       "      <td>10.002</td>\n",
+       "      <td>5.231</td>\n",
+       "      <td>HI UH LOOKS LIKE THE CALL IS IN APARTMENT THRE...</td>\n",
```
[the remark this answers is not in the document]
Comment (ShiyangLai) [reply pipelines/notebooks/bpc_vad_error.ipynb:131]: I will remove it now.

**Pull request merged**